### PR TITLE
P0 cart management

### PR DESCRIPTION
**Pull request by purvasharmadev**
<h2>Major Changes:</h2>
<li> Added Cart Page </li>

kstyle-cart-page

<h2>Minor Changes</h2>
<li> Fix the navbar </li>
<li> Added some utility classes </li>
<li> Added a colour variable</li>
<li> Wrote a discount class </li>

### CODE REVIEW

File: cart-management/cart.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    color: var(--GRAY-COLOR);
+}
+
+
+.cart-card{
+    height: fit-content;
+    width: 30rem;
+    align-items: center;
+    background-color: var(--TEXT-COLOR);
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
```
Comment (zoef-yt) [inline cart-management/cart.css]: Use var for colour

**Review by zoef-yt — APPROVED** [review]
Nice small pr. The pr description is really good helps with the review. The code is good. The UI also looks great. Good job. Good to merge.

**Review by rutu14 — APPROVED** [review]

**Review by zoyatanweer — APPROVED** [review]
clean code, good to merge

File: cart-management/cart.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+h2{
+    letter-spacing: 2px;
+    color: var(--PRIMARY-COLOR);
+}
+
+h3{
+    color: var(--GRAY-COLOR);
+}
+
+
```
Comment (adityapandadev) [inline cart-management/cart.css]: Use Uniform Spacing

File: cart-management/cart.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            <li class="nav-item"><input type="text" class="icon" placeholder="search here"></li>
+        </ul>
+        <ul class="nav-list">
+            <li class="nav-item"><a href="/wishlist.html" class="nav-link"><span class="badge badge-span">
+                        <i class="fa fa-heart fa-x"></i>
+                        <span class="badge-circle badge-right"> 2 </span>
+                    </span></a></li>
+            <li class="nav-item"><a href="/cart.html" class="nav-link link"><span class="badge badge-span">
+                        <i class="fa fa-cart-plus fa-x"></i>
+                        <span class="badge-circle badge-right"> 0 </span></span> Cart
```
Comment (adityapandadev) [inline cart-management/cart.html:30]: It will be good, if you could write normal text in new line , separated from html codes

Comment (purvasharmadev) [reply cart-management/cart.html:30]: Thank you for the suggestion, will keep in mind.

**Review by adityapandadev — APPROVED** [review]
Good to go, good job 👍.

Well formatted code structure

**Pull request merged**